integration-tests/oidc-wiremock/README.md

```diff
@@ -0,0 +1,25 @@
+# JAX-RS example using Keycloak adapter to protect resources
```

Comment:
Please drop this README file, these wiremock tests will be running with `mvn clean install` :-). I will also follow up with the documentation.